scripts/observers/config.py

```diff
@@ -0,0 +1,113 @@
+import os
+import toml
+import sys
+import shutil
+from pathlib import Path
+from git import Repo
+from dotenv import load_dotenv
+
+load_dotenv()
+
+observer_dir = str(os.getenv('OBSERVER_DIR'))
+METACHAIN = 4294967295
+
+
+def update_toml_indexer(path, shard_id):
+    # prefs.toml
+    data1 = toml.load(path+"/prefs.toml")
```

Comment:
If path is a `pathlib.Path`, then you can also use the `/` operator. For example: https://github.com/ElrondNetwork/elrond-sdk-images/blob/main/scripts/build_contract_rust_within_docker.py#L104

Reply:
refactored.